frontend/src/components/VoteResultChart.css

```
@@ -35,6 +29,18 @@
   display: none;
 }
 
+.vote-result-chart__text {
+  padding: 2px;
+}
+
+.vote-result-chart__text--for {
+  background: var(--green);
+}
+
+.vote-result-chart__text--against {
+  background: var(--red);
+}
```

You can avoid adding extra classes to the markup if you use selectors like this:

```suggestion
.vote-result-chart__bar--for .vote-result-chart__text {
  background: var(--green);
}

.vote-result-chart__bar--against .vote-result-chart__text {
  background: var(--red);
}
```